src/services/base.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import elasticsearch.exceptions
+from elasticsearch import AsyncElasticsearch
+
+from db.cache import ModelCache
+
+logger = logging.getLogger(__name__)
+
+
+class BaseESService:
```

Comment:
Можно лучше: Кажется, если BaseESService нельзя использовать напрямую без наследования, то стоит его сделать абстрактным классом, что думаешь?
```suggestion
class BaseESService(ABC):
```